src/osx/webview_webkit.mm

```diff
@@ -408,13 +411,46 @@ @interface WebViewCustomProtocol : NSURLProtocol
     return wxCFStringRef::AsString([dr toString]);
 }
 
-void wxWebViewWebKit::RunScript(const wxString& javascript)
+bool wxWebViewWebKit::RunScript(const wxString& javascript, wxString* output)
 {
     if ( !m_webView )
-        return;
+    {
+        wxLogWarning(_("wxWebView is not created!"));
```

Comment:
This _is_ a programmer error and not something the user needs to know about, so it should be an assert. I.e. replace the entire check with

    wxCHECK_MSG( m_webView, false, wxS("wxWebView must be created before running JS scripts") );